model/annotations/task_annotations.go

```diff
@@ -210,3 +218,36 @@ func UpdateAnnotation(a *TaskAnnotation, userDisplayName string) error {
 	)
 	return errors.Wrapf(err, "problem adding task annotation for '%s'", a.TaskId)
 }
+
+func GetJiraTickets(issuelinks []IssueLink) ([]DisplayTicket, error) {
+	var displayTickets []DisplayTicket
+
+	settings := evergreen.GetEnvironment().Settings()
+	jiraHandler := thirdparty.NewJiraHandler(*settings.Jira.Export())
+
+	for _, issue := range issuelinks {
+		displayTicket := DisplayTicket{}
+
+		urlObject, err := url.Parse(issue.URL)
+		if err != nil {
+			return nil, errors.Wrap(err, "problem parsing issue string")
```

Comment:
If the URL is malformed, we'll never display a ticket, and since they can be user-inputted (I think) it's not impossible. It might be better to catch these errors and log them, rather than blocking the whole process (a malformed URL could still go through the 236 block).

Reply:
If the url is malformed, I won't be able to access url.Host. Are you saying that I should catch the error and still get the tickets for the rest of the tickets in the array? 

Alternatively, would it make more sense to add a check to the addIssue graphql mutation?